CHANGES.rst

```diff
@@ -1,6 +1,13 @@
 CHANGES
 =======
 
+1.1.0 (2019/08/02)
+------------------
+
+Bug Fixes:
```

Comment:
I think this is a environment issue instead of a bug.
Until now we don't have such category in the changes.rst, .. how do you think?

Reply:
Ummm... I'll removing group title :)

Reply:
We did not decide the rules, I think it is good...